### PR TITLE
fix content assist in user preferences

### LINKED ISSUE #4661: content assist does not work in preferences widget

### PR DESCRIPTION
**Pull request by akosyakov**
fix #4661

### CODE REVIEW

**Comment by vince-fugnitto** [comment]
@akosyakov I'm sorry but can you briefly explain what the content assist is and how I can test the patch ?

**Comment by akosyakov** [comment]
> @akosyakov I'm sorry but can you briefly explain what the content assist is and how I can test the patch ?

If you open user preferences editor, and press ctrl + space, you should get proposals for preference names. It is not the case on master.

**Comment by vince-fugnitto** [comment]
> > @akosyakov I'm sorry but can you briefly explain what the content assist is and how I can test the patch ?
> 
> If you open user preferences editor, and press ctrl + space, you should get proposals for preference names. It is not the case on master.

thank you ! I'll test the PR now 😃

File: packages/json/src/browser/json-client-contribution.ts [hunk 1]
```diff
@@ -67,7 +67,7 @@ export class JsonClientContribution extends BaseLanguageClientContribution {
         for (const s of allConfigs) {
             if (s.fileMatch) {
                 for (let fileMatch of s.fileMatch) {
-                    if (fileMatch.charAt(0) !== '/' && !fileMatch.match(/\w+:\/\//)) {
+                    if (fileMatch.charAt(0) !== '/' && !fileMatch.match(/\w+:/)) {
```
Comment (akosyakov) [inline packages/json/src/browser/json-client-contribution.ts]: @svenefftinge would be fine to remove them?

Comment (svenefftinge) [reply packages/json/src/browser/json-client-contribution.ts]: It's fine for now but might pop up again, once we use the vs code extension.

**Pull request merged**